src/app/beer_garden/db/mongo/util.py

```diff
@@ -266,10 +211,10 @@ def check_indexes(document_class):
         existing = document_class._get_collection().index_information()
 
         if document_class == Request and "parent_instance_index" in existing:
-            raise OperationFailure("Old Request index found, rebuilding")
+            raise OperationError("Old Request index found, rebuilding")
```

Comment:
Not your fault here, but the idea that we explicitly raise exceptions from another package seems weird to me.  Catching and re-raising, sure.  But raising those exceptions ourselves doesn't seem appropriate.  This really ought to be a custom exception defined in `beer_garden.errors`, or alternatively this could be refactored so that the conditional checks control the code flow, rather than the try / except.

Reply:
I can add a new error to our custom ones if that works. I sort of assume that goes in `brewtils.errors` unless there's an error file in the beergarden directory I'm not seeing.